tools/stonith_admin.c

```diff
@@ -323,31 +323,15 @@ handle_level(stonith_t *st, char *target, int fence_level,
                                        name, value, fence_level);
 }
 
-static char *
-fence_action_str(const char *action)
-{
-    char *str = NULL;
-
-    if (action == NULL) {
-        str = strdup("unknown");
-    } else if (action[0] == 'o') { // on, off
-        str = crm_concat("turn", action, ' ');
```

Comment:
There was a slight difference in the two versions: this one would say "turn on", "turn off", while the other would say "unfencing", "turning off" (which is what the new one goes with). That means this now prints something like

rhel7-3 was able to turning off node node3 on behalf of stonith_admin.5703

which besides being funny breaks the fence regression tests :)

Unfortunately the crm_mon usage has to stay as it is. Easiest thing will be to just keep this variant here, alternatively you could add an argument to stonith_action_str() to select the sense.

Reply:
See #1733